src/react-components/create-object-dialog.js

```diff
@@ -50,7 +50,8 @@ export default class CreateObjectDialog extends Component {
   onUrlChange = e => {
     let attributionImage = this.state.attributionImage;
     if (e.target && e.target.value && e.target.validity.valid) {
-      attributionImage = attributionHostnames[new URL(e.target.value).hostname];
+      const url = new URL(e.target.value);
```

Comment:
Not an issue, but was this change intended to be kept?

Reply:
🤷‍♂️ Maybe a readability thing. I'll keep it.